### PR TITLE
jazz params in

### PR DESCRIPTION
**Pull request by itajaja**

### CODE REVIEW

File: src/webdriver/index.ts [hunk 1]
```diff
@@ -98,11 +98,14 @@ export function augmentDriver<T>(driver: WebDriver, baseUrl: string) {
         })
         .perform();
     },
+    
+    // type ValidType = string | boolean | Record<string, ValidType> | ValidType[]
```
Comment (AaronBuxbaum) [inline src/webdriver/index.ts:102]: ? how does this work with this commented out?

Comment (itajaja) [reply src/webdriver/index.ts:102]: I just dropped this a s a sketch real quick, I'd like to get to it, I think should be easy

Comment (AaronBuxbaum) [reply src/webdriver/index.ts:102]: might want to put in a comment about what it's for, otherwise it looks like dead code

**Review by AaronBuxbaum — APPROVED** [review]

File: src/webdriver/index.ts (hunk 1 above)
Comment (itajaja) [reply src/webdriver/index.ts:102]: I am saying, I need to work on this pr. i just put some very early draft on :P. I literally modified some files to explain what I had in mind and pushed

**Pull request closed without merging**